thirdparty/icu4c/APIChangeReport.md

```diff
@@ -5,381 +5,265 @@
  License & terms of use: http://www.unicode.org/copyright.html
 -->
 
-# ICU4C API Comparison: ICU 67 with ICU 68
+# ICU4C API Comparison: ICU 68 (update #1: 68.2) with ICU 69
```

Comment:
I think we could remove that file, usually we only include the files we compile + license/copyright details.

Reply:
Removed.